data/infusory.mdx

```diff
@@ -0,0 +1,14 @@
+---
+name: Infusory
+description: We provide next reality immersive solutions with the latest in tech, for industries and educational institutes.
+founded: Jan 2018
+based: Both
+domain: Computer And Related Activities
+logo: infusory-logo.png
+location: Kochi
+address: RISE - CUSATECH Foundation Student Amenity Centre CUSAT, Kalamassery, Kochi, Kerala 682022
+website: https://infusory.in/
+email: info@infusory.in
+linkedin: https://www.linkedin.com/company/aioncw/
```

Comment:
@libinmv linked in url not of `infusory`

Reply:
Resolved, good eye!!